src/main/java/org/zalando/nakadi/repository/zookeeper/ZookeeperProxy.java

```diff
@@ -0,0 +1,121 @@
+package org.zalando.nakadi.repository.zookeeper;
+
+import org.apache.zookeeper.CreateMode;
+import org.apache.zookeeper.KeeperException;
+import org.apache.zookeeper.ZooDefs;
+import org.apache.zookeeper.ZooKeeper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+/**
+ * Non thread safe implementation of coordination service backed by Apache Zookeeper.
+ */
+public class ZookeeperProxy implements CoordinationService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ZookeeperProxy.class);
+    private static final int RETRY_DELAY_MS = 250;
+    private static final int RETRY_MAX_COUNT = 10;
+    private final ZooKeeper zookeeper;
+
+    public ZookeeperProxy(final ZooKeeper zookeeper) {
+        this.zookeeper = zookeeper;
+        try {
+            zookeeper.create("/nakadi/locks", null, ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
```

Comment:
if you are creating `/nakadi/locks` here, than path in lockAccuire should be relative to this path. I don't see it.